unity/Assets/Scripts/StretchAgentController.cs

```diff
@@ -268,6 +268,73 @@ public partial class StretchAgentController : ArmAgentController {
             actionFinished(true);
         }
 
+//        public void RotateCameraBase(float yawDegrees, float rollDegrees) {
+//            var target = gimbalBase;
+//            var maxDegree = maxBaseXYRotation;
+//            Debug.Log("yaw is " + yawDegrees + " and roll is " + rollDegrees);
+//            if (yawDegrees < -maxDegree || maxDegree < yawDegrees) {
+//                throw new InvalidOperationException(
+//                    $"Invalid value for `yawDegrees`: '{yawDegrees}'. Value should be between '{-maxDegree}' and '{maxDegree}'."
+//                );
+//            } else if (rollDegrees < -maxDegree || maxDegree < rollDegrees) {
+//                throw new InvalidOperationException(
+//                    $"Invalid value for `rollDegrees`: '{rollDegrees}'. Value should be between '{-maxDegree}' and '{maxDegree}'."
+//                );
+//            } else {
+//                gimbalBase.localEulerAngles = new Vector3(
+//                    gimbalBaseStartingXRotation + rollDegrees,
+//                    gimbalBaseStartingYRotation + yawDegrees,
+//                    gimbalBase.transform.localEulerAngles.z
+//                );
+//            }
+//            actionFinished(true);
+//        }
+
+        public void RotateCameraMount(float degrees, bool secondary = false) {
```

Comment:
Do we need this funtionality on this branch?

Reply:
it shows up here probably because I merged the code from `physics-det-new-cam-adjust`
shall I remove this?

Reply:
I see I think we can remove it